src/uu/sort/src/sort.rs

```diff
@@ -196,6 +246,13 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         "Sort by SemVer version number, eg 1.12.2 > 1.1.2",
     );
     opts.optflag("c", "check", "check for sorted input; do not sort");
+    opts.optopt("", "head", "output a maximal number of elements", "INTEGER");
```

Comment:
oh, if you are motivated, in a different PR, would be great to migrate this to clap :)